server/src/main/java/org/elasticsearch/common/settings/Settings.java

```diff
@@ -506,9 +493,7 @@ public Version getAsVersion(String setting, Version defaultVersion) throws Setti
      */
     public String toDelimitedString(char delimiter) {
         StringBuilder sb = new StringBuilder();
-        for (Map.Entry<String, Object> entry : settings.entrySet()) {
-            sb.append(entry.getKey()).append("=").append(entry.getValue()).append(delimiter);
-        }
+        settings.forEach((key, value) -> sb.append(key).append("=").append(value).append(delimiter));
```

Comment:
This appears to be an unrelated formatting change. I’m also doubting it, since now we have those lambda allocations in a loop. I’ve personally never been a fan of using `forEach` to mutate, I’ve felt it should be side-effect free (because it looks like an expression, not a statement). Either way, it’s unrelated to our main story.